src/main/groovy/com/smartbear/swagger/assertion/ValidStatusCodesAssertionCreator.groovy

```diff
@@ -0,0 +1,16 @@
+package com.smartbear.swagger.assertion
+
+import com.eviware.soapui.impl.wsdl.teststeps.RestTestRequestStep
+import com.eviware.soapui.security.assertion.ValidHttpStatusCodesAssertion
+import io.swagger.models.Response
+
+import static com.eviware.soapui.security.assertion.ValidHttpStatusCodesAssertion.LABEL
+
+
+class ValidStatusCodesAssertionCreator implements AssertionCreator {
+    @Override
+    void createAssertion(RestTestRequestStep restTestRequestStep, Map<String, Response> responseMap) {
+        ValidHttpStatusCodesAssertion assertion = (ValidHttpStatusCodesAssertion) restTestRequestStep.addAssertion(LABEL)
+        assertion.codes = responseMap.keySet().join(',')
```

Comment:
I think Swagger has the concept of a "default" response code - we need to filter these out - see http://swagger.io/specification/#responsesDefault